docs/references/javascript/user/password-management.mdx

```diff
@@ -1,39 +1,282 @@
 ---
 title: User Password Management
-description: Learn about the methods on the User object that help you manage a user's password.
+description: Explore methods on the User object that help you manage a user's password.
 ---
 
 # User password management
 
-The [`User`](/docs/references/javascript/user/user) object has methods that help you manage a user's password.
+These methods on the [`User`](/docs/references/javascript/user/user) object help you manage a user's password.
+
+The following examples assume that you have followed the [quickstart](/docs/quickstarts/javascript) in order to add Clerk to your JavaScript application.
 
 ## `updatePassword()`
 
+Updates the user's password. Note that passwords must be at least 8 characters long.
+
 ```typescript
 function updatePassword: (params: UpdateUserPasswordParams) => Promise<User>;
 ```
 
-Updates the user's password.
-
 ### `UpdateUserPasswordParams`
 
-
 | Name | Type | Description |
 | --- | --- | --- |
 | `newPassword` | `string` | The user's new password. |
-| `currentPassword?` | `string` | The user's current password. |
+| `currentPassword` | `string` | The user's current password. |
 | `signOutOfOtherSessions?` | `boolean \| undefined` | If set to `true`, all sessions will be signed out. |
 
+### `updatePassword()` example
+
+<InjectKeys>
+
+<Tabs type="npm-script" items={['NPM module', '<script>']}>
+  <Tab>
+    <CodeBlockTabs options={["main.js", "index.html"]}>
```

Comment:
I think it's better to split the `main.js` snippet and `index.html` out instead of having them be two tabs on the same codeblock. I don't think it's immediately obvious that you're supposed to use both to build the full example, and a lot of our docs use code block tabs to show alternatives rather than separate but equally necessary snippets

Reply:
We do use codeblocktabs to show multiple files - an example is [here](https://docs-preview-802.clerkpreview.com/docs/advanced-usage/satellite-domains#configure-your-satellite-app-with-properties)

If you believe this shouldn't be the case, we can create a ticket to fix instances in the docs that do this.

Or,
I can add some text that says something like "The tabs below demonstrate what your JavaScript and HTML files could look like when using ClerkJS's NPM module."

What do you think we should do?

Reply:
I think for now you can add that text, and in future I think we should avoid this pattern. Rather than trying to fix them all at once in a single ticket, let's just be mindful that when we touch these snippets we should break them up. That way it can be done over time

Reply:
And why do you think we should avoid this pattern? If we add copy to clearly indicate what the tabs are for, then why not utilize our custom components to reduce visual noise/page length?

Reply:
It's not consistent design language. Many places in our docs, different tabs = alternative options, eg "App router" and "Pages router". Having copy to indicate things is good, but people skim the docs and expect our code snippets to be structured generally the same way throughout.

Reply:
Yes